frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -3,21 +3,233 @@ import { Scoped } from "kremling";
   }
+];
+
+export default function MotionToWaiveFees_Web({ data }) {
+  return (
+    <Scoped css={css}>
+      <FormThatPrints>
+        <h1 className="web-from-input">This is a private record</h1>
+        <Section name="1. Personal information">
+          <TextInput dataKey="person.firstName" label="First name" />
+          <TextInput dataKey="person.middleName" label="Middle name" />
+          <TextInput dataKey="person.lastName" label="Last name" />
+          <TextInput dataKey="person.address" label="Street address" />
+          <TextInput dataKey="person.city" label="City" />
+          <TextInput dataKey="person.state" label="State" />
+          <TextInput dataKey="person.zip" label="Zip" />
+          <TextInput dataKey="person.phone" label="Phone" />
+          <TextInput dataKey="person.email" label="Email" />
+          <small className="web-form-input">
+            Check your email. You will receive information and documents at this
+            email address.
+          </small>
+
+          <Select
+            dataKey="person.relation"
+            label="Relation"
+            options={waiveFeeRelationship}
+          />
+
+          {data.person.relation &&
+            data.person.relation.includes("Attorney") && (
+              <TextInput label="Bar Number" dataKey="person.attorney" />
+            )}
+        </Section>
+        <Section name="2. Case Information">
+          <Radio
+            dataKey="case.courtType"
+            label="Court Type"
+            options={courtTypeOptions}
+          />
+
+          {data.case.courtType === "District" && (
+            <GroupSelect
+              dataKey="case.courtAddress"
+              label="District Courts"
+              groupOptions={DistrictCourtList}
+            />
+          )}
+          {data.case.courtType === "Justice" && (
+            <GroupSelect
+              dataKey="case.courtAddress"
+              label="Justice Courts"
+              groupOptions={JusticeCourtList}
+            />
+          )}
+
+          <TextInput dataKey="case.number" label="Case number" />
+          <TextInput dataKey="case.judge" label="Judge" />
+          <TextInput dataKey="case.commissioner" label="Commissioner" />
+          <small className="web-form-input">(Domestic cases)</small>
+        </Section>
+        <Section name="3. Fees to waive">
+          <p className="web-form-input">
+            I ask for the following fee(s) to be waived:
+          </p>
+          <small className="web-form-input">(Choose all that apply)</small>
+          <CheckBox dataKey="case.filingFee" label="Filing fee" />
+          <small className="web-form-input">(Refer to Civil Cover Sheet)</small>
+          {data.case.filingFee && (
+            <TextInput
+              label="Filing Fee Amount"
+              dataKey="case.filingFeeAmount"
+            />
+          )}
+          <CheckBox dataKey="case.ocapFee" label="OCAP fee" />
```

Comment:
These checkboxes reminded me that we should really have the CheckBox component require that you pass in a `name` prop. The name prop groups checkboxes for the same question together.

Can you add that to the CheckBox component. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-name

Reply:
Will do.